packages/wdio-types/src/Capabilities.ts

```diff
@@ -89,10 +90,36 @@ export interface MultiRemoteCapabilities {
 }
 
 export type RemoteCapability = DesiredCapabilities | W3CCapabilities | MultiRemoteCapabilities;
-export interface Worker extends Omit<TestrunnerOptions, 'capabilities'> {
+
+export interface Job {
+    caps: DesiredCapabilities | W3CCapabilities | MultiRemoteCapabilities
+    specs: string[],
+    hasTests: boolean
+}
+
+export type WorkerMessageArgs = Omit<Job, 'caps' | 'specs' | 'hasTests'>;
+
+export interface WorkerRunPayload {
```

Comment:
I don't think `WorkerRunPayload` is in any way connected to capabilities. I suggest to have them directly in `packages/wdio-types/src/index.ts`